packages/js/src/generators/typescript-sync/typescript-sync.spec.ts

```diff
@@ -14,6 +14,7 @@ let projectGraph: ProjectGraph;
 jest.mock('@nx/devkit', () => ({
   ...jest.requireActual('@nx/devkit'),
   createProjectGraphAsync: jest.fn(() => Promise.resolve(projectGraph)),
+  formatFiles: jest.fn(() => Promise.resolve()),
```

Comment:
Making `formatFiles` noop still produces the same results in the tests' expectations proving the changes produce the correct formatting.